docs/exercises/5-multi_region.rst

```diff
@@ -334,10 +337,12 @@ key to illustrate that encryption and decryption is still possible by using a ke
 of the supported resource based access control mechanisms that allow you to programmatically delegate the use of CMKs.
 Grants enable more granular permissions management.
 
-In this portion of the exercise, we will be adding a grant to the local key that checks if the encryption context
-equals {'key use':'bad key'}. Upon local key retrieval, this grant will block this key from being accessed because
-the encryption context supplied with the ciphertext will not match this constraint. Therefore, preventing the local
-key from being accessed.
+In this portion of the exercise, we will be adding a grant to the local key in us-east-2 that will block the key from
+being accessed. In this grant, we will be adding an encryption context constraint that will only permit use of the key
+if the encryption context matches the one specified in the grant explicitly. Since we do not want to permit access to
```

Comment:
"encryption context constraint that requires a specific encryption context that our application is not using"?